specs/multispec.yml

```diff
@@ -156,6 +156,15 @@ specs:
       c9s_image_name: "sclorg/postgresql-{{ spec.short }}-c9s"
       latest_fedora: "f38"
 
+    "16":
+      version: "16"
+      prev_version: "13"
+      short: "16"
+      prev_short: "13"
```

Comment:
Same as above, there should be 15.

Reply:
Fixed by https://github.com/sclorg/postgresql-container/pull/548/commits/5fa4da9c64afffa0d29b3e4e4ca5a6cad541d5c7